documentation/secure_storage_sql.md

```diff
@@ -0,0 +1,77 @@
+# SQL DB Secure Storage
+
+## Introduction
+
+This document describes the SQL DB secure storage in OP-TEE, which is enabled
```

Comment:
"Which" is better. I'm not sure if being a native speaker really helps here. After all, we have a popular grammar book "Elements of Style" which is full of incorrect rules, including concerning which/that.